deno/lib/benchmarks/competitive.ts

```diff
@@ -0,0 +1,85 @@
+import Benchmark from "benchmark";
+import mz from "myzod";
```

Comment:
I expect to see an update to `package.json` that adds `myzod` to `devDependencies`, no?

Reply:
ugh, this was included by accident - see #1059 for that issue. i’ll remove it when i’m back at a keyboard

Reply:
Removed the stray file.